server/odc-core/src/test/java/com/oceanbase/odc/core/sql/execute/mapper/MySQLGeometryMapperTest.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.oceanbase.odc.core.sql.execute.mapper;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.ResultSetMetaData;
+import java.sql.Statement;
+
+import javax.sql.DataSource;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.oceanbase.odc.test.database.TestDBConfigurations;
+import com.oceanbase.tools.dbbrowser.model.datatype.JdbcDataTypeFactory;
+
+/**
+ * @author jingtian
+ * @date 2023/11/21
+ * @since ODC_release_4.2.4
+ */
+public class MySQLGeometryMapperTest {
+    private DataSource dataSource;
+    private final MySQLGeometryMapper mapper = new MySQLGeometryMapper();
+
+    @Before
+    public void setUp() throws Exception {
+        dataSource = TestDBConfigurations.getInstance().getTestOBMysqlConfiguration().getDataSource();
+        prepareEnv();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        destroyEnv();
+    }
+
+    @Test
+    public void test_POINT_datatype() throws Exception {
```

Comment:
we'd better put only one assert in a test case

Reply:
done